GVRf/Framework/jni/engine/importer/jassimp.cpp

```diff
@@ -36,6 +36,24 @@
 
 namespace gvr {
 
+class DeleteLocalRef {
+private:
+    JNIEnv* mEnv;
+    jobject& mObject;
+public:
+    DeleteLocalRef(JNIEnv* env, jobject& object) : mEnv(env), mObject(object) {};
+    DeleteLocalRef(JNIEnv* env, jclass& clazz) : mEnv(env), mObject((jobject&) clazz) {};
+    DeleteLocalRef(JNIEnv* env, jfloatArray& farray) : mEnv(env), mObject((jobject&) farray) {};
+    DeleteLocalRef(JNIEnv* env, jintArray& iarray) : mEnv(env), mObject((jobject&) iarray) {};
+    DeleteLocalRef(JNIEnv* env, jstring& str) : mEnv(env), mObject((jobject&) str) {};
+    ~DeleteLocalRef() {
+        if (mObject != NULL) {
+            mEnv->DeleteLocalRef(mObject);
+        }
+    }
+};
+
```

Comment:
If willing to go through the hassle to create a separate header you might consider making DeleteLocalRef a template class. It will yield fewer lines of code.